.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 - [ ] All [tests](https://github.com/jfrog/jfrog-client-go#tests) passed. If this feature is not already covered by the tests, I added new tests.
+- [ ] All [static analysis checks](https://github.com/jfrog/jfrog-client-go/actions/workflows/analysis.yml) passed.
```

Comment:
I'd prefer adding instructions for running the static checks in the readme, like we do for the tests. 
```suggestion
- [ ] All [static analysis checks](https://github.com/jfrog/jfrog-client-go#static-check) passed.
```

Reply:
Will add in the future as discussed